workspaces/libnpmpack/test/fixtures/tspawk.js

```diff
@@ -5,6 +5,7 @@ const spawk = require('spawk')
 module.exports = tspawk
 
 function tspawk (t) {
+  spawk.load()
```

Comment:
Ah now I remember why `tspawk(t)` was called using the top level tap and not inside tests.

I moved it inside the test when I was debugging the last PR, and I think it should be moved back.

Can you remove this, and move the `const spawk = tspawk(t)` back to the top of the test file right after the `require(spawk)`?

Reply:
Try it first and if it requires removing we remove it, it's alright.

Reply:
I recall it being required since other tests spawn processes that wouldn't be matched (that's why I removed it originally) -- I'll double check, though.